src/test/java/at/grahsl/kafka/connect/mongodb/converter/RecordConverterTest.java

```diff
@@ -82,6 +84,8 @@ public static void initializeTestData() {
                                     .build())
                 )
                 .field("mySubDoc2", SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.INT32_SCHEMA).build())
+                .field( "myMapOfStrings", SchemaBuilder.map(Schema.STRING_SCHEMA, SchemaBuilder.array(Schema.STRING_SCHEMA).build()).build())
```

Comment:
hi again. sorry it took me some time to get back on this...
I'm really looking forward to merging this. Let me suggest one more small aspect which is that I think for consistency reasons of this test we should also add the two maps to the JSON_STRING_1 above at right before the "myBytes" field 

see https://github.com/hpgrahsl/kafka-connect-mongodb/pull/86/commits/0c0ad67cc850f8d9f219941a6b9c84e0edc9101d#diff-ab414b445c07f6db6f112afe2ca89aafL63

could you maybe please do it directly in your branch? then it's the least effort I guess to get it merged :)